fastai/text/models/awdlstm.py

```diff
@@ -63,7 +72,10 @@ def _do_nothing(self): pass
 class EmbeddingDropout(Module):
     "Apply dropout with probability `embed_p` to an embedding layer `emb`."
 
-    def __init__(self, emb, embed_p):
+    def __init__(self,
+        emb:nn.Embedding, # Wrapped embedding layer
+        embed_p:float # Embdedding <word> dropout probability
```

Comment:
Yeah, it would be nice to make it clear that specific input ids embedding vectors are zeroed entirely rather then individual embedding vector elements. So I thought that would be intuitive: like we apply dropout to "words" in vocab. Guess both phrasing are not quite perfect